SCJ.Booking.CourtBookingBulkEmailService/Program.cs

```diff
@@ -0,0 +1,62 @@
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Graph.ExternalConnectors;
+using SCJ.Booking.MVC.Data;
+using SCJ.Booking.MVC.Models;
+using SCJ.Booking.MVC.Services;
+using Serilog;
+
+namespace SCJ.Booking.CourtBookingBulkEmailService
+{
+    public class Program
+    {
+        private static ILogger _logger;
+        private static MailService _mailService;
+        private static ApplicationDbContext _dbContext;
+
+        public Program(ApplicationDbContext dbContext, IConfiguration configuration)
+        {
+            _logger = LogHelper.GetLogger(configuration);
+            _dbContext = dbContext;
+            _mailService = new MailService("CA", configuration, _logger);
+        }
+
+        public static void Main(string[] args)
+        {
+            //get the successful court booking emails in batches of 50 to send out
+            var courtBookingEmailBatch = _dbContext.Set<CourtBookingEmail>().Take(50);
+
+            try
+            {
+                foreach (var email in courtBookingEmailBatch)
+                {
+                    string body = "";
+                    if (email.BodyText.Length > 0 && email.BodyHtml.Length > 0)
+                        body = email.BodyHtml + email.BodyText; //unsure of how to combine them atm
+                    else if (email.BodyText.Length > 0)
+                        body = email.BodyText;
+                    else if (email.BodyHtml.Length > 0)
+                        body = email.BodyHtml;
+
+                    //update the sender email address
+                    _mailService.ChangeSenderEmail(email.CourtLevel);
+                    _mailService.ExchangeSendEmail(email.ToEmail, email.Subject, body).Wait();
```

Comment:
it's probably better for threading to make the whole thing async than to use `.Wait()` .  
see https://github.com/bcgov/workbc-jb/blob/master/src/WorkBC.Importers.Federal/Program.cs for an example of an async service.

Reply:
Ah perfect, I briefly looked into making the Main function async but VS complained.  I'll update to be async.